src/components/pages/kurtApp/KurtApp.tsx

```diff
@@ -0,0 +1,16 @@
+import { Outlet } from 'react-router-dom'
+import NavBar from '../../templates/NavBar'
+import { Footer } from '../../organisms/Footer/Footer'
+import { useMemo } from 'react'
+
+export function KurtApp() {
+  const navBar = useMemo(() => <NavBar />, [])
+  const footer = useMemo(() => <Footer />, [])
+  return (
+    <>
+      {navBar}
+      <Outlet />
```

Comment:
What does the outlet give us? Is that just going to be replaced with the child component?

Reply:
Exactly. It's how Router knows where to put the next component.